traefik/plan.sh

```diff
@@ -0,0 +1,68 @@
+pkg_name=traefik
+pkg_description="a modern reverse proxy"
+pkg_upstream_url="https://traefik.io"
+pkg_origin=core
+pkg_version="v1.3.6"
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("MIT")
+pkg_source="http://github.com/containous/traefik"
+pkg_build_deps=(
+  core/yarn
+)
+pkg_deps=(
+  core/node # not needed at runtime, but fix_interpreter requires this
+            # https://github.com/habitat-sh/habitat/issues/1041
+)
+pkg_bin_dirs=(bin)
+pkg_svc_user="root"
+pkg_svc_group="root"
+pkg_scaffolding=core/scaffolding-go
```

Comment:
So at the moment we're in the process of rewriting the go-scaffolding. From looking through this plan it looks like theres not a whole lot of the scaffolding functionality being used here. 

I think it might be better _for now_ to rip this out rather than leverage it for a core plan today. Relying on this right now might by dangerous since the changes we're planning could be breaking.